src/app/pages/client-reviews/client-reviews.component.ts

```diff
@@ -0,0 +1,87 @@
+import { Subscription } from 'rxjs';
+import { Component, OnInit, Input } from '@angular/core';
+import { ClientReviewsService } from 'src/app/services/client-reviews/client-reviews.service';
+import { ActivatedRoute, Router } from '@angular/router';
+import { APPCONFIG } from 'src/app/config';
+import { ToastrService } from 'ngx-toastr';
+import { NgxSpinnerService } from 'ngx-spinner';
+
+@Component({
+  selector: 'app-client-reviews',
+  templateUrl: './client-reviews.component.html',
+  styleUrls: ['./client-reviews.component.scss']
+})
+export class ClientReviewsComponent implements OnInit {
+  subscription: Subscription = new Subscription();
+  id: number;
+  count: number;
+  review: string;
+  reviewer: any;
+  createdAt: string;
+  replies: Array<any>[];
+  results: any[];
```

Comment:
Great work @dorothyas , Could you kindly show me where you are using id, replies and review , I seem to not see where they are being used.

Reply:
I have worked on this